### PR TITLE
Hanad 003 feature

### PR DESCRIPTION
**Pull request by hahmed377**

### CODE REVIEW

**Comment by hahmed377** [comment]
Indented and attempts field added

File: index.js [hunk 1]
```diff
@@ -1,79 +1,94 @@
...
+        $('.battleShip').removeClass('o');
+        test = 5;
+        for (var i = 0; i < minClick; i++) {
+          $('')
+        }
+        $('.battleShip').click(function(event){
+          score += 10;
+          $(".score").html("score: " + score);
+          alert("hit you  " + score);
+          })
```
Comment (AEJackson) [inline index.js]: need ; at end of call to click

File: index.html [hunk 2]
```diff
@@ -10,22 +10,23 @@
...
+      <button id="Instructions" name="button">Instructions</button>
+      <button id="Startgame" name="button">Start game</button>
+      <h2 class='Instructions'></h2>
+      <div class="row begin-countdown">
+        <div class="text-center">
+          <p class="start"> Begining in <span id="pageBeginCountdownText">10 </span> seconds</p>
+          <p class="score">score</p>
+          <p class = "attempts">attempts</p>
+          <p class = "message" id="messageArea"></p>
+      </div>
```
Comment (AEJackson) [inline index.html]: end div's miss-aligned
look at formatting / indentation

**Comment by hahmed377** [comment]
I've added features, comments and it is indented

File: index.js [hunk 3]
```diff
@@ -1,79 +1,187 @@
...
+          }
+          $(this).addClass("x");
+        // this will add the class x after the ship is clicked on
+      });
+    };
+  }
+  // this will retrieve the item from local storage and added to a list.
+  function displayScore() {
+    for (var i = 1; i < 3; i++) {
+    $(".leaderboards").append("<li>"+localStorage.getItem(i)+"</li>");
```
Comment (AEJackson) [inline index.js]: check indentation

**Pull request merged**